integration-tests/actions/vrfv2plus/vrfv2plus_steps.go

```diff
@@ -43,90 +48,94 @@ var (
 	ErrGetPrimaryKey            = "error getting primary ETH key address"
 	ErrRestartCLNode            = "error restarting CL node"
 	ErrWaitTXsComplete          = "error waiting for TXs to complete"
+
+	ErrRequestRandomness             = "error requesting randomness"
+	ErrWaitRandomWordsRequestedEvent = "error waiting for RandomWordsRequested event"
+
+	ErrWaitRandomWordsFulfilledEvent = "error waiting for RandomWordsFulfilled event"
 )
 
 func DeployVRFV2PlusContracts(
 	contractDeployer contracts.ContractDeployer,
 	chainClient blockchain.EVMClient,
+	consumerContractsAmount int,
 ) (*VRFV2PlusContracts, error) {
 	bhs, err := contractDeployer.DeployBlockhashStore()
 	if err != nil {
 		return nil, errors.Wrap(err, ErrDeployBlockHashStore)
 	}
+	//err = chainClient.WaitForEvents()
```

Comment:
do we not need this? if no, good to remove?

Reply:
enabled back the waitforevents